src/vm/codeversion.cpp

```diff
@@ -1457,7 +1457,7 @@ HRESULT MethodDescVersioningState::UpdateJumpTarget(BOOL fEESuspended, PCODE pRe
         *pnOffset = offset;
 
         hr = UpdateJumpStampHelper(pbCode, i64OldValue, i64NewValue, !fEESuspended);
-        _ASSERTE(hr == S_OK || (hr == CORPROF_E_RUNTIME_SUSPEND_REQUIRED && !fEESuspended));
+        _ASSERTE(hr == S_OK || ((unsigned int)hr == CORPROF_E_RUNTIME_SUSPEND_REQUIRED && !fEESuspended));
```

Comment:
Can you please instead modify the definition of the CORPROF_E_RUNTIME_SUSPEND_REQUIRED by wrapping the constant in _HRESULT_TYPEDEF_ macro?
And the same for other places where you have added cast of hr.

Reply:
done